### PR TITLE
vmm:qemu support aarch64

### PR DESCRIPTION
**Pull request by HITweihailiuxu**
vmm:qemu support aarch64

### CODE REVIEW

File: vmm/sandbox/src/qemu/config.rs [hunk 1]
```diff
@@ -21,7 +21,7 @@ use std::{
 };
 
 use containerd_sandbox::error::{Error, Result};
-#[cfg(target_arch = "x86_64")]
+#[cfg(any(target_arch = "x86_64", target_arch = "aarch64"))]
```
Comment (Burning1020) [inline vmm/sandbox/src/qemu/config.rs:24]: Is this cfg not needed anymore after aarch64 is supportted? If so, you can remove it.

**Review by Burning1020 — CHANGES_REQUESTED** [review]

File: vmm/sandbox/src/qemu/config.rs (hunk 1 above)
Comment (HITweihailiuxu) [reply vmm/sandbox/src/qemu/config.rs]: done

File: vmm/sandbox/src/qemu/config.rs [hunk 2]
```diff
@@ -40,24 +40,34 @@ const DEFAULT_QEMU_PATH: &str = "/usr/bin/qemu-system-x86_64";
...
 lazy_static! {
     static ref SUPPORTED_MACHINES: HashMap<String, Machine> = {
         let mut sms = HashMap::new();
+        #[cfg(target_arch = "x86_64")]
         sms.insert(
             "microvm-pci".to_string(),
             Machine {
                 r#type: "microvm-pci".to_string(),
```
[the remark this answers is not in the document]
Comment (HITweihailiuxu) [reply vmm/sandbox/src/qemu/config.rs]: done

File: vmm/sandbox/src/qemu/config.rs [hunk 3]
```diff
@@ -155,17 +165,17 @@ impl QemuVMConfig {
         } else {
             return Err(Error::InvalidArgument("cpu model".to_string()));
         }
-        #[cfg(target_arch = "x86_64")]
+        #[cfg(any(target_arch = "x86_64", target_arch = "aarch64"))]
```
[the remark this answers is not in the document]
Comment (HITweihailiuxu) [reply vmm/sandbox/src/qemu/config.rs]: done

**Review by Burning1020 — APPROVED** [review]
LGTM. Thanks

**Pull request merged**